web/templates/settings/invites_list_admin.hbs

```diff
@@ -8,7 +8,12 @@
     <div class="settings_panel_list_header">
         <h3>{{t "Invitations "}}</h3>
         <div class="alert-notification" id="invites-field-status"></div>
-        <input type="text" class="search filter_text_input" placeholder="{{t 'Filter invitations' }}" aria-label="{{t 'Filter invitations' }}"/>
+        <div class="search_container">
+            <input type="text" class="search filter_text_input" placeholder="{{t 'Filter invitations' }}" aria-label="{{t 'Filter invitations' }}"/>
+            <button type="button" class="clear_filter">
+                <i class="zulip-icon zulip-icon-close"></i>
+            </button>
+        </div>
```

Comment:
So it seems costly add this much duplicated code for each widget. I think you can probably create a `web/templates/settings/filter_text_input.hbs`  partial that takes a few parameters including the label, and just include that partial in each of these places rather than copying the code into each one.